src/main/java/threads/MyThread.java

```diff
@@ -0,0 +1,16 @@
+package threads;
+
+import org.apache.log4j.Logger;
+
+public class MyThread extends Thread {
+    private static final int COUNT = 100;
```

Comment:
two threads should increment common counter ( you should pass it from outside into constructor) that would be thread race